build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml

```diff
@@ -1,4 +1,6 @@
 parameters:
+  BuildForHelix: 'true'
+  conditionForDiscoverTests: ''
   jobName: 'RunTestsInHelix'
   dependsOn: ''
   condition: ''
```

Comment:
Ah, GitHub won't let me comment on the line, but we'll also need to update Line 28 to use the new version of the Nuget package. 
```
winUIHelixVer: 0.5.4
```

And the file `test\WindowsAppSDK.Helix.TestCommon\WindowsAppSDK.Helix.TestCommon.csproj` should be updated as well. (Line 14).

```
    <PackageReference Include="Microsoft.Internal.WinUI.Helix" Version="0.5.4" GeneratePathProperty="true" />
```

Reply:
Doh!

Reply:
Grep finds 2 references to 0.5.4

* build\AzurePipelinesTemplates\WindowsAppSDK-RunHelixTests-Job.yml
* test\WindowsAppSDK.Helix.TestCommon\WindowsAppSDK.Helix.TestCommon.csproj

Updated both to 0.5.5